spec/models/user_spec.rb

```diff
@@ -45,35 +45,32 @@
   end
 
   describe 'legacy_check_password' do
-    before(:all) do
-      @sf_user = create(:sf_guard_user, salt: 'SALT', password: Digest::SHA1.hexdigest('SALTPEANUTS'))
-      @user = create(:user, username: 'unique', sf_guard_user_id: @sf_user.id)
-    end
+    let(:sf_user) { create(:sf_guard_user, salt: 'SALT', password: Digest::SHA1.hexdigest('SALTPEANUTS')) }
+    let(:user) { create(:user, username: 'unique', sf_guard_user_id: sf_user.id) }
+
     it 'returns true for correct password' do
-      expect(@user.legacy_check_password('PEANUTS')).to be true
+      expect(user.legacy_check_password('PEANUTS')).to be true
     end
 
     it 'returns false for incorrect password' do
-      expect(@user.legacy_check_password('FAKE_PEANUTS')).to be false
+      expect(user.legacy_check_password('FAKE_PEANUTS')).to be false
     end
   end
 
   describe 'create_default_permissions' do
-    before do
-      @sf_user = create(:sf_guard_user, username: "user#{rand(1000)}")
-      @user = create(:user, sf_guard_user_id: @sf_user.id, email: "#{rand(1000)}@fake.com")
-    end
+    let(:sf_user) { create(:sf_guard_user, username: "user#{rand(1000)}") }
+    let(:user) { create(:user, sf_guard_user_id: sf_user.id, email: "#{rand(1000)}@fake.com") }
 
     it 'creates contributor permission' do
-      expect(@user.has_legacy_permission('contributor')).to be false
-      @user.create_default_permissions
-      expect(@user.has_legacy_permission('contributor')).to be true
+      expect(user.has_legacy_permission('contributor')).to be false
```

Comment:
out of curiosity: curious why you wound up re-working these tests? because they caught your attention while you were figuring out permissions for `recent_edits`?

Reply:
I wanted to write couple of testes for user -- i opened the user_spec file and didn't like i what i saw, so i did something about it :)